test/spec/oauthUtil.js

```diff
@@ -1,6 +1,54 @@
 define(function(require) {
   var OktaAuth = require('OktaAuth');
   var oauthUtil = require('../../lib/oauthUtil');
+  var util = require('../util/util');
+  var wellKnown = require('../xhr/well-known');
+
+  describe('getWellKnown', function() {
+    util.itMakesCorrectRequestResponse({
```

Comment:
Here it might be better to just test that "we get well known, and use the cached version when it's stored", i.e. the end-to-end where we can verify that we got the same response both times, but only made a request once